subsys/bluetooth/controller/ll_sw/ull_llcp_internal.h

```diff
@@ -516,7 +526,8 @@ void llcp_lr_pause(struct ll_conn *conn);
 void llcp_lr_resume(struct ll_conn *conn);
 void llcp_lr_tx_ack(struct ll_conn *conn, struct proc_ctx *ctx, struct node_tx *tx);
 void llcp_lr_tx_ntf(struct ll_conn *conn, struct proc_ctx *ctx);
-void llcp_lr_rx(struct ll_conn *conn, struct proc_ctx *ctx, struct node_rx_pdu *rx);
+void llcp_lr_rx(struct ll_conn *conn, struct proc_ctx *ctx, memq_link_t *link,
+		struct node_rx_pdu *rx);
```

Comment:
Wrong alignment or is i just diff that is weird?

Reply:
Hmm, indent by two tabs, seems ok?